Content.Server/Blocking/BlockingSystem.cs

```diff
@@ -0,0 +1,72 @@
+using Content.Server.Examine;
+using Content.Shared.Blocking;
+using Content.Shared.Damage;
+using Content.Shared.Damage.Prototypes;
+using Content.Shared.Verbs;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Utility;
+
+namespace Content.Server.Blocking;
+
+public sealed class BlockingSystem : SharedBlockingSystem
+{
+    [Dependency] private readonly ExamineSystem _examine = default!;
+    [Dependency] private readonly IPrototypeManager _proto = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<BlockingComponent, GetVerbsEvent<ExamineVerb>>(OnVerbExamine);
+    }
+
+    private void OnVerbExamine(EntityUid uid, BlockingComponent component, GetVerbsEvent<ExamineVerb> args)
+    {
+        if (!args.CanInteract || !args.CanAccess)
+            return;
+
+        _proto.TryIndex<DamageModifierSetPrototype>(component.PassiveBlockDamageModifer, out var passiveblockModifier);
+        _proto.TryIndex<DamageModifierSetPrototype>(component.ActiveBlockDamageModifier, out var activeBlockModifier);
```

Comment:
```suggestion
    var activeBlockModifier = component.ActiveBlockDamageModifier;
    var passiveblockModifier = component.PassiveBlockDamageModifer;
```

Instead of calling _proto.TryIndex you should be able to do this change and then make modifications to the review below. If you need an example check out Server.ArmorSystem line 70.